### PR TITLE
Making it working !

### PR DESCRIPTION
**Pull request by Aeris1One**
TODO : 

- [x] Fix Install
- [x] Fix Removing
- [x] Fix Upgrading
- [ ] Fix Backup
- [ ] Fix Restoring
- [ ] Fix Package Linter
- [X] Fix Manifest (thanks @ericgaspar)

Build Status&build=15)

### CODE REVIEW

**Comment by yalh76** [comment]
Courage, j'avais commencé et au bout de quelques heures j'ai abandonné ^^

File: scripts/install [hunk 1]
```diff
@@ -30,169 +30,123 @@ path_url=$YNH_APP_ARG_PATH
...
 #=================================================
 
 #ynh_replace_string "random_key" "$random_key" "$final_path/.env"
-ynh_replace_string "yunouser"   "$db_name"    "$final_path/.env"
-ynh_replace_string "yunopass"   "$db_pwd"     "$final_path/.env"
-ynh_replace_string "yunobase"   "$db_name"    "$final_path/.env"
-ynh_replace_string "yunomail"   "$email"      "$final_path/.env"
-ynh_replace_string "yunodomain" "$domain"     "$final_path/.env"
+ynh_replace_string "__DB_USER__"   "$db_name"    "$final_path/.env"
+ynh_replace_string "__DB_PASS__"   "$db_pwd"     "$final_path/.env"
```
Comment (yalh76) [inline scripts/install:122]:
```suggestion
ynh_replace_string "__DB_PWD_"   "$db_pwd"     "$final_path/.env"
```

File: scripts/install [hunk 2]
```diff
@@ -30,169 +30,123 @@ path_url=$YNH_APP_ARG_PATH
...
+ynh_replace_string "__DB_NAME__"   "$db_name"    "$final_path/.env"
```
Comment (yalh76) [inline scripts/install:123]: You should use https://yunohost.org/fr/packaging_apps_helpers?q=%2Fpackaging_apps_helpers
`ynh_replace_string --match_string=match_string --replace_string=replace_string --target_file=target_file`

File: scripts/install [hunk 3]
```diff
@@ -203,14 +157,26 @@ ynh_package_install docker-ce
...
+
 mkdir -p /srv/daemon /srv/daemon-data
 pushd /srv/daemon
-curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
-ynh_package_install nodejs make gcc g++
-curl -L https://github.com/pterodactyl/daemon/releases/download/v0.6.11/daemon.tar.gz | tar --strip-components=1 -xzv
+
+ynh_install_nodejs --nodejs_version=10
+
+ynh_package_install make gcc g++
```
Comment (yalh76) [inline scripts/install:170]: You should add that in _common.sh, in `pkg_dependencies`

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Comment by yalh76** [comment]
!testme

**Comment by Aeris1One** [comment]
By the power of systemd, I invoke The Great App CI to test this Pull Request!

**Comment by Aeris1One** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by Aeris1One** [comment]
What about those two open PRs ? Should #35 be merged into this one ?

**Comment by imnotjack91** [comment]
i want to help but i dont have much experience with this app.
I am also a relatively new yunohost user but i want to contribute. 
This app is important to me and i would like to see it make it to stable. 
where can i begin helping?

**Pull request closed without merging**